packages/react-components/source/react/library/icon/icons.js

```diff
@@ -1029,7 +1029,20 @@ const icons = {
       />
     ),
   },
-
+  'plus-circle-filled': {
+    medium: (
+      <>
+        <path d="M7.75 15.9575C12.0302 15.9575 15.5 12.4877 15.5 8.20752C15.5 3.92731 12.0302 0.45752 7.75 0.45752C3.46979 0.45752 0 3.92731 0 8.20752C0 12.4877 3.46979 15.9575 7.75 15.9575Z" />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          fill="white"
```

Comment:
We also usually remove all the `fill` attributes so that icons can be styled with CSS. However, I can see in this case that the icon was constructed by overlaying a white plus over a solid circle, so simply removing it wouldn't work in this case. I'm just wondering whether or not we should have a designer construct the icon using negative space instead.

Reply:
I have a new export of a flattened icon that should address @vine77 's last comment. 

The new code should be a single path. Note that I have not scrubbed this for correctly matching format - its the raw code from Figma.

`<path fill-rule="evenodd" clip-rule="evenodd" d="M8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16ZM7 7V4H9V7H12V9H9V12H7V9H4V7H7Z" />`

Reply:
Here it is as a separate file:

[Plus-Circle-Solid 16px.zip](https://github.com/puppetlabs/design-system/files/6774498/Plus-Circle-Solid.16px.zip)